wiki/Modding/BAT/en.md

```diff
@@ -0,0 +1,26 @@
+tags:
+  - BAT
+  
+# The Beatmap Appreciation Team 
+
+The Beatmap Appreciation Team (also called the Beatmap Approval Team, or BAT) were a precursor group to the [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) and Quality Assurance Team, <!-- TODO: QAT stub --> responsible for overseeing the beatmap [Ranking and Disqualification](/wiki/Beatmap_ranking_procedure) processes. Additionally, BAT members possessed basic tools for in-game and forum moderation also used by the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) (GMT).
+  
+Together with the [Mapping Assistance team](/wiki/Modding/MAT) (MAT) until their teams merged in April 2013, the Beatmap Appreciation Team were responsible for evaluating the quality of beatmaps and deciding what content made its way into the Ranked <!-- TODO: Ranked status stub --> section.
+
+In addition to co-operation with the Mapping Assistance Team, from the 15th of May, 2013 onwards, the BAT were regulated by a group of [BAT Managers](/wiki/Modding/BATManagers) (BATm) who oversaw their operation and provided guidance where necessary.
+
+## History of the BAT
+
+The Beatmap Appreciation Team was formed as early as 2008. <!-- TODO: Much more info required here -->
+
+Prospective Beatmap Appreciation Team members could apply to join by submitting a sample of their modding activity and experience during allocated cycles, and would be qualitatively reviewed by existing BAT members (and later also the BAT Managers) for their suitability as a BAT member. The frequency of these application cycles was determined by the BAT Managers. Alternatively, the BAT Managers could opt to directly invite users with exceptional potential.
+
+On the 21st of August, 2014, the Beatmap Appreciation Team was split into the Beatmap Nomination Group (later the [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators)) and the Quality Assurance Team <!-- TODO: QAT stub -->. From this point onwards, the Beatmap Nomination Group would focus on the process of nominating Beatmaps for the Ranked category, whilst the Quality Assurance Team would instead focus on checking these Qualified maps, and  disqualifying <!-- TODO: DQ stub --> any they deemed to be breaking the [Ranking Criteria](/wiki/Ranking_Criteria), or otherwise lacking in quality.
+
+Existing members of the Beatmap Appreciation Team at this time were given the option of remaining in the BAT to focus on beatmap qualification (giving up their moderation powers and responsibilities in the process), or transitioning to the Quality Assurance Team, who retained these moderation powers and responsibilities.
+
+From the 31st of January onwards, the group was renamed the "Beatmap Appreciators", (and shortly thereafter the "Beatmap Nomination Group"). At this time, the rules on admission of new members to the BAT were changed to allow entry via application only on a monthly basis.
```

Comment:
```suggestion
From the 31st of January onwards, the group was renamed the "Beatmap Appreciators", and shortly thereafter the "Beatmap Nomination Group". At this time, the rules on admission of new members to the BAT were changed to allow entry via application only on a monthly basis.
```

if you were to remove the part currently in parenthesis you'd get `From the 31st of January onwards, the group was renamed the "Beatmap Appreciators", .` which makes no sense. I don't think the parenthesis are really needed around the second half of the sentence either, so best to just remove.

Reply:
Fixed with clayton's suggestions